ultralytics/data/explorer/explorer.py

```diff
@@ -466,3 +466,77 @@ def generate_report(self, result):
         TODO
         """
         pass
+
+    def label_count(self, labels: list) -> dict:
+        """
+        Count of Occurrences of labels
+        Args:
+            labels (list): List of labels to count their occurrences.
+
+        Returns:
+            label_counts(dict): Dictionary containing labels with their count
+
+        Example:
+            ```python
+                exp = Explorer()
+                exp.create_embeddings_table()
+                label_counts = exp.label_count()
+            ```
+        """
+        label_counts = {}
+        base_query = "SELECT * FROM 'table' WHERE"
+        for i in range(len(labels)):
+            if i == 0:
+                base_query += f" labels LIKE '%{labels[i]}%'"
```

Comment:
this will count "person, person, person" and "person" as one label right? That isn't right?

Reply:
No, It will count each and every occurrence